packages/core/src/codecs/octetstream-codec.ts

```diff
@@ -115,23 +136,24 @@ export default class OctetstreamCodec implements ContentCodec {
 
     private integerToValue(
         bytes: Buffer,
-        options: { dataLength: number; bigendian: boolean; signed: boolean }
+        options: { dataLength: number; bigEndian: boolean; signed: boolean }
     ): number {
-        const { dataLength, bigendian, signed } = options;
+        const { dataLength, bigEndian, signed } = options;
+
         switch (dataLength) {
-            case 1:
+            case 8:
```

Comment:
Out of curiosity. What is the big win using bit dataLength vs. byte dataLength?

Reply:
In the `integerToValue` method, the only point is to be consistent with using bit length throughout the codec where possible.
Further up at line 105, when reading only the bits specified at the bit dataLength, I am ensuring that the bit offset is a multiple of 8, so I could use the byte dataLength here, too.